docs/content/guides/operator/snapshots.mdx

```diff
@@ -7,13 +7,31 @@ Database snapshots provide a point-in-time view of a database's store. In Sui, t
 
 Snapshots of the Sui network enable Full node operators a way to bootstrap a Full node without having to execute all the transactions that occurred after genesis. You can upload snapshots to remote object stores like S3, Google Cloud Storage, Azure Blob Storage, and similar services. These services typically run the export process in the background so there is no degradation in performance for your Full node. With snapshots stored in the cloud, you're more easily able to recover quickly from catastrophic failures in your system or hardware.
 
+To maintain a healthy Sui network, Sui encourages the Sui community to bring up additional snapshots to ensure stronger data availability across the network.
+
+## Supported snapshot types
+
+Sui supports two types of snapshots:
+
+- **RocksDB snapshots** are a point-in-time view of a database store. This means that the snapshot keeps the state of the system at the moment it generates the snapshot, including non-pruned data, additional indices, and other data.
+- **Formal snapshots** are database agnostic: they keep the state of the system in the minimalistic format. The Formal snapshots also include cryptographic state accumulators, which are certified by the quorum of the validators. These can be verified by the node during the reconstruction. This is one of the main advantages over the RocksDB snapshots, which don't include any special integrity data certified by a quorum.
```

Comment:
```The Formal snapshots also include cryptographic state accumulators, which are certified by the quorum of the validators. These can be verified by the node during the reconstruction.```
Do we want to word it a little bit like - "The formal snapshots are verified during restore using cryptographic state accumulation by comparing against the state root hash from the verified end of epoch checkpoint summary". Also cc @williampsmith for suggestions if any

Reply:
Absolutely, I just wanted us to have a conversation about what we all think should be in the docs or if it needs to be tweaked a little bit. Your suggestion sounds great!